### PR TITLE
LRQA-81189 | master

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://liferay.atlassian.net/browse/LRQA-81189

tested on release bundle https://github.com/catarineaguiar/liferay-portal/pull/102

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/clustering/clusteringframework/Clustering.testcase [hunk 1]
```diff
@@ -422,6 +416,7 @@ definition {
 	@priority = 4
 	test CanUpdateLogLevelsForAllNodesFromSlave {
 		property app.server.bundles.size = "2";
+		property custom.properties = "license.cluster.overload.node.auto.shut.down=false";
```
[the remark this answers is not in the document]
Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/clustering/clusteringframework/Clustering.testcase:419]: @catarineaguiar

[the remark this answers is not in the document]
Comment (catarineaguiar) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/clustering/clusteringframework/Clustering.testcase:419]: oh, sorry for that, I was having issues to open the third node and was put this property by mistake. I'll remove it. Thanks for catching this @zhangyan0701

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
LGTM! Thanks!

See https://github.com/vicnate5/liferay-portal/pull/2404.

**Pull request closed without merging**